### PR TITLE
Add python 3.7 to tox and appveyor test matrix

### PR DESCRIPTION
**Pull request by klette**
Tests will fail until #17 is merged

### CODE REVIEW

**Comment by miketheman** [comment]
Thanks! Can you rebase from `master` and push again to get the tests running?
Also, please add this version to the `classifiers` section in `setup.py`:

https://github.com/miketheman/pytest-socket/blob/a00cda374176d841b7cf942a2c54d1b6a1c37a90/setup.py#L36-L38

**Comment by klette** [comment]
Done. Had to change the travis-setup a bit to support it. Travis is not really that up to speed lately :-)

File: .travis.yml [hunk 1]
```diff
@@ -4,14 +4,18 @@ env:
...
 language: python
+
 python:
   - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
-  - "pypy"
+  - "3.7"
+  - "pypy2.7-5.10.0"
```
Comment (miketheman) [inline .travis.yml]: Should we be using pypy 2.7 here? From the docs, it appears that we ought to use pyp3.5:
https://docs.travis-ci.com/user/languages/python/#pypy-support

**Comment by miketheman** [comment]
I ended up dropping pypy support, thanks for trying!

**Pull request closed without merging**